cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -389,6 +389,52 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    SelfPacedDueDateEditor = AbstractEditor.extend({
+        fieldName: 'due_num_weeks',
+        templateName: 'self-paced-due-date-editor',
+        className: 'modal-section-content has-actions due-date-input grading-due-date',
+
+        events: {
+            'click .clear-date': 'clearValue',
+            'keyup #due_in': 'validateDueIn'
+        },
+
+        getValue: function() {
+            return parseInt(this.$('#due_in').val());
+        },
+
+        validateDueIn: function() {
+            if (parseInt(this.$('#due_in').val()) > 18){
```

Comment:
Nit: consistent in how you access the value. Either using `this.getValue` everywhere (like yo do inside of `getRequestData` or with the `parseInt` and dropping the `getValue` function. Happy with either solution, just want to consistently apply it